### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by luizfroes**
### Work Done
- 1 pages HTML project with about me, contact and projects and a nav bar with links for each section
- used CSS for styling
- updated readme with all information and screenshots

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+body {
+  height: 100%;
+  width: 100%;
+  background-image: url("../images/html-images/background-image.png");
+  background-size: cover;
+  background-position: center;
+  font-family: "Ubuntu", sans-serif;
+}
+
+/* This is the class selector for class name of header */
```
Comment (tashamann) [inline assets/css/style.css]: Great to see you including comments but they don't need to be this detailed - you can simply just write `/* header section */` for example. Go through and edit all your comments so they just tell us which section of the HTML the styles are referring to.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+/* This the pseudo class hover applied on the contact class */
+.contact:hover {
+  text-decoration: underline;
+  font-size: 1.4rem;
+}
+
+/* tablet properties */
+@media screen and (max-width: 768px) and (min-width: 577px) {
+  .nav-conteiner,
```
Comment (tashamann) [inline assets/css/style.css]: You have a typo here so this style won't be applied.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+  }
+
+  .secondary-cards {
+    width: 45%;
+  }
+}
+
+/* mobile properties */
+@media screen and (max-width: 576px) {
+  .nav-conteiner {
```
Comment (tashamann) [inline assets/css/style.css]: Another typo

File: index.html [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Luiz Froes</title>
+  </head>
+
+  <body>
+    <!--This is the header with the nav bar-->
+    <header class="header">
+      <!--This is the Navigation bar with the portfolio website links-->
```
Comment (tashamann) [inline index.html]: Again, your comments in this file can be much more brief too. Eg `<!--Navigation bar-->`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!--This is the Footer containing the Contacts section -->
+    <footer class="footer" id="contact">
+      <ul class="contacts-list">
+        <li class="contact">Email</li>
```
Comment (tashamann) [inline index.html:233]: Please add in the contact info here! Check out the `mailto` option on the `href` attribute to allow the link to directly open up the user's email. You can also add your linkedin link too.

File: assets/css/style.css (hunk 1 above)
Comment (luizfroes) [reply assets/css/style.css]: Done!!

File: assets/css/style.css (hunk 2 above)
Comment (luizfroes) [reply assets/css/style.css]: Done!!

File: assets/css/style.css (hunk 3 above)
Comment (luizfroes) [reply assets/css/style.css]: Done!!

File: index.html (hunk 4 above)
Comment (luizfroes) [reply index.html]: All done!!

File: index.html (hunk 5 above)
Comment (luizfroes) [reply index.html:233]: I will add in the future as i dont have a Linkedin yet and i dont wanna expose my email for now.

thanks

**Pull request merged**